src/satosa/micro_services/account_linking.py

```diff
@@ -52,6 +53,10 @@ def _handle_al_response(self, context):
             satosa_logging(logger, logging.INFO, "issuer/id pair is linked in AL service",
                            context.state)
             internal_response.user_id = message
+            id_to_attr = self.config.get("id_to_attr", None)
```

Comment:
`config` is static and set at initialisation time through `__init__`. I propose we move this to `__init__` too, as `id_to_attr` is not going to change and calling `.get` over and over is only wasting cpu cycles.

```diff
@@ -32,6 +32,7 @@ def __init__(self, config, *args, **kwargs):
          self.redirect_url = config["redirect_url"]
          self.signing_key = RSAKey(key=rsa_load(config["sign_key"]), use="sig", alg="RS256")
          self.endpoint = "/handle_account_linking"
+         self.config = config
+         self.id_to_attr = self.config.get("id_to_attr", None)
          logger.info("Account linking is active")
  
      def _handle_al_response(self, context):
```

then all you need to do is
```python
if self.id_to_attr:
    internal_response.attributes[self.id_to_attr] = [message]
```